packages/algob/package.json

```diff
@@ -51,6 +51,14 @@
     "publish": "npm publish --access public",
     "algob:dev": "sh ./setup_dev_project.sh"
   },
+  "files": [
+    "src/",
```

Comment:
do we need `src`? `build` already has `.d.ts` files.

Reply:
i don't think so. 
let me know if you want to exclude it from package ?

Reply:
removing src as well